charts/longhorn/101.0.0+up1.2.6/README.md

```diff
@@ -0,0 +1,62 @@
+# Longhorn Chart
```

Comment:
This `README.md` file contains wrong content. It shouldn't contains instruction for helm and kubectl. Seems like the some patching were incorrect? 

A correct one should look like https://github.com/rancher/charts/blob/dev-v2.6/charts/longhorn/100.1.4%2Bup1.2.6/README.md

Reply:
I regenerated the chart files again and the README file is now consistent with dev-2.6.